### DISCUSSION TITLE
Default board selection mpfs-icicle-kit-es

### DISCUSSION THREAD

**Discussion by Mendandukka** [polarfire-soc/polarfire-soc-documentation#271] [HSS and U-Boot] [answered]
Hi,

I am using RCHD-PF-EVAL board designed by Conclusive Engineering. I want to try some base design project by using SoftConsole (v2022.2). I renamed rchd-pf boards def_config file to .config and paste it to hss file for specified board and initialize system. But everytime i have INFO about "BOARD not specified, defaulting to mpfs-icicle-kit-es". 

Is there any solution for change the default board maybe or why i couldn't describe my board is not icicle kit ?

Thanks in advance for your help.

**Comment by TommyMurphyTM1234** [comment, marked as answer]
I presume that you're referring to the HSS?
That being the case, the message is coming from here:

* https://github.com/polarfire-soc/hart-software-services/blob/8af865f74810f216329606be7721da939239e68a/Makefile#L50

and you need to specify your custom board name via the `BOARD` env var/parameter and make sure that all references via this variable resolve correctly.
